style.css

```diff
@@ -0,0 +1,13 @@
+#navbar {
+    position: fixed;
+    top: 0;
+  }
+  
+  #welcome-section {
+    height: 100vh
+  }
+  @media (max-width: 768px) {
+    body{
+      background-color: brown;
+    }
+  }
```

Comment:
- [ ] Good job pointing out the use of line indentations 👍. However, it is good to include an explanation as to why a change is required with reasons why you think they should make the changes, as it will help the developer to see the importance of the change, and help them be a better developer. Please, let us edit this comment to be to include why using a particular line indentation is a good practice.

Reply:
Kindly a new line after line 13 which is the last line in your css page..
Not doing so will be the wrong use of line indentation.
This will enable any browser interpret your code well.